backend/.env

```diff
@@ -1 +1,2 @@
-PORT=8000
\ No newline at end of file
+PORT=8000
```

Comment:
Make sure you are giving right path for `.env` file in `.gitignore`. Update it with correct path and do remove `.env` from github.